### PR TITLE
set sizingTarget, simplify demo

### LINKED ISSUE #206: Demo does not work for scrolling overlay in landscape mode
The demo running on the polymer elements web site.
The overlay does not centre vertically when in landscape.

Environment context:
Android Nexus 6, latest standard android.
Chrome browser

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #206

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -120,57 +120,39 @@
...
+          <div class="tall">scrollable content</div>
         </div>
-        <p>
-          <button onclick="scrolling2.close()">Close</button>
-        </p>
-
+        <button onclick="scrolling2.close()">Close</button>
       </simple-overlay>
+
+      <script>
```
Comment (bicknellr) [inline demo/index.html]: Does this get executed when the cloned template contents are inserted? How does it look in Safari / Firefox?

Comment (valdrinkoshi) [inline demo/index.html]: Tested on both browsers and looks fine, `sizingTarget` is correctly set
screen shot 2016-07-19 at 12 11 34 pm
screen shot 2016-07-19 at 12 11 05 pm

Comment (bicknellr) [inline demo/index.html]: alright, thanks

**Comment by bicknellr** [comment]
There's no longer any indication that the element in the overlay scrolls. Is there some way we could make it apparent?

**Comment by valdrinkoshi** [comment]
Uhm, something like changing "scrollable content" text to "scroll me"?

**Comment by valdrinkoshi** [comment]
@bicknellr, I made scrollable area more visible:
screen shot 2016-07-20 at 11 01 57 am
screen shot 2016-07-20 at 11 01 48 am

**Comment by bicknellr** [comment]
LGTM, sorry about asking for the text back but it's much easier to see what's going on when all of that junk is in there!

**Comment by valdrinkoshi** [comment]
np, it's indeed clearer now ;)

**Pull request merged**